Data/DataModel/Event.cs

```diff
@@ -29,19 +29,70 @@ public Uri URL
         public bool IsInternEvent { get; set; }
         public DateTime TeamRegistrationBegin { get; set; }
         public DateTime TeamRegistrationEnd { get; set; }
+
+        /// <summary>
+        /// Returns whether or not team registration is active.
+        /// </summary>
+        /// <returns>True if the current date is between the team registration begin and end times.</returns>
+        public bool IsTeamRegistrationActive()
```

Comment:
Can we make these into standard gettable properties instead? Note that you will need the [NotMapped] attribute if they become properties.